mmv1/third_party/terraform/tests/resource_compute_router_bgp_peer_test.go

```diff
@@ -195,6 +238,7 @@ func testAccComputeRouterPeerBasic(routerName string) string {
 	return fmt.Sprintf(`
 resource "google_compute_network" "foobar" {
   name = "%s-net"
+  auto_create_subnetworks = false
```

Comment:
All these examples actually create a subnet, so we disable auto subnet creation to speed things up and not face subnet quota issues.